projects/observability/src/shared/components/cartesian/cartesian-chart.component.ts

```diff
@@ -57,6 +57,9 @@ export class CartesianChartComponent<TData> implements OnChanges, OnDestroy {
   @Input()
   public timeRange?: TimeRange;
 
+  @Input()
+  public isSelectionHandlerAvailable: boolean = false;
```

Comment:
```suggestion
  public isRangeSelectionEnabled: boolean = false;
```

1. handler available gets too much into the details of how it works than what behavior we're trying to control
2. the name of selection is too generic, since charts can feasibly have other types of selections, like clicking on a bar).